cmd/commands/cli/command_cli.go

```diff
@@ -53,6 +53,8 @@ const statusConnected = "Connected"
 
 // Run runs CLI interface synchronously, in the same thread while blocking it
 func (c *Command) Run() (err error) {
+	license := cmd.GetStartupLicense("type `show w'", "type `show c'")
```

Comment:
GetStartupLicense() - I don't see it defined. I would guess it should return error (file not found.. etc).

Reply:
It was defined in `license.go`, but it was too long file to be displayed in PR, so it was collapsed.

Split that file so that code would be separate from long texts.